src/node/chain_spec.rs

```diff
@@ -1,7 +1,9 @@
 //! Subspace chain configurations.
 
 use sc_service::ChainType;
-use sc_subspace_chain_specs::{ChainSpecExtensions, SerializableChainSpec};
+#[cfg(feature = "executor")]
+use sc_subspace_chain_specs::ChainSpecExtensions;
+use sc_subspace_chain_specs::SerializableChainSpec;
```

Comment:
why is this not under `#[cfg(feature = "executor")]`
could find anywhere else this is being used without the executor context?

Reply:
Didn't get it. `ChainSpecExtensions` are needed only when we have system domain, so that is why it is under `cfg(executor)`. As for `SerializableChainSpec` we will still use it even if we disable executor (for ser/de chain spec)

Reply:
I know, but in this specific file `chain_spec.rs`, `SerializableChainSpec` is only used when `executor` flag is there (at least to my eye).
So I thought, maybe it would be better to also make that conditional

Reply:
It is used in both branches actually :eyes: